src/index.ts

```diff
@@ -1298,10 +1298,21 @@ async function claimLinkSender({
 
 	// Prepare transaction options
 	let txOptions = {}
-	txOptions = await setFeeOptions({
-		txOptions,
-		provider: signer.provider,
-	})
+	try {
+		txOptions = await setFeeOptions({
+			provider: structSigner.signer.provider,
+			eip1559: structSigner.eip1559,
+			maxFeePerGas: structSigner.maxFeePerGas,
+			maxPriorityFeePerGas: structSigner.maxPriorityFeePerGas,
+			gasLimit: structSigner.gasLimit,
+		})
+	} catch (error) {
+		throw new interfaces.SDKStatus(
+			interfaces.ESignAndSubmitTx.ERROR_SETTING_FEE_OPTIONS,
+			error,
+			'Error setting the fee options'
+		)
+	}
```

Comment:
![image](https://github.com/peanutprotocol/peanut-sdk/assets/12943235/f0638515-0f88-4b6e-8d25-a5074297fad3)

I think the way we defined SDKStatus, error and 'Error setting the fee options' should be switched in order, no?

Maybe this makes more sense to switch around in the SDKStatus object, instead of here